pages/who-we-are/index.js

```diff
@@ -34,6 +35,7 @@ const WhoWeAre = () => {
           ))}
         </div>
       </div>
+      <Timeline />
```

Comment:
I would do something like this for now to get closer to what we want:

```jsx
<div className="h-screen flex">
        <div className="w-1/2" /> // placeholder for left side
        <div className="w-1/2 overflow-y-scroll">
          <Timeline />
        </div>
</div>
```

and then create an issue to work on the placeholder. Also move this block above the rest so its at the top

Reply:
I have another PR for the other side coming pretty quick so if its okay, let's skip this and review the next PR.